sites/authentication/authentication-en.html

```diff
@@ -4,58 +4,61 @@
 	"breadcrumb": {
 		"title": "Canada.ca", "link": "https://www.canada.ca/en.html"
 	},
-	"dateModified": "2022-05-17",
+	"dateModified": "2022-08-24",
 	"description": "Documentation on how to use the various authentication patterns.",
 	"language": "en",
 	"title": "Authentication patterns"
 }
 ---
 <p>This component support the following 3 design pattern variant:</p>
 <ul>
-	<li>Contextual Sign-in button: A link pointing to a dedicated sign-in web page</li>
-	<li>Signed-off: A pattern that show how the user can sign-on directly from the current page. Note: The prompt sign-in dialog and the interaction resulting of the signed-on are currently out-of-scope.</li>
+	<li>Contextual Sign in button: A link pointing to a dedicated sign in web page</li>
+	<li>Signed-off: A pattern that show how the user can sign-on directly from the current page. Note: The prompt sign in dialog and the interaction resulting of the signed-on are currently out-of-scope.</li>
 	<li>Signed-on: A pattern that shows the user is logged in, a way to access your account and how to disconnect.</li>
 </ul>
 
-<h2>Contextual Sign-in button</h2>
+<h2>Contextual Sign in button</h2>
```

Comment:
Is there two (2) independent sub-component with their own version number or is it one (1) component but with two (2) version.